hack/defaults

```diff
@@ -83,16 +88,23 @@ function ssp_sed {
     sed -i "s|          image: \&image quay\.io\/fromani\/kubevirt-ssp-operator-container:${SSP_VERSION}|          image: \&image ${SSP_CONTAINER_REGISTRY}\/kubevirt-ssp-operator-container:${SSP_VERSION}|g" ${TEMP_DIR}/operator.yaml
 }
 
+function vm_import_sed {
+    sed -i "s|          image: \&image quay\.io\/kubevirt\/kubevirt-vm-import-operator-container:${VM_IMPORT_VERSION}|          image: \&image ${VM_IMPORT_CONTAINER_REGISTRY}\/kubevirt-vm-import-operator-container:${VM_IMPORT_VERSION}|g" ${TEMP_DIR}/operator.yaml
+}
+
 # Deploy upstream operators
 OPERATOR_MANIFESTS="https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-operator.yaml
 https://github.com/kubevirt/containerized-data-importer/releases/download/${CDI_VERSION}/cdi-operator.yaml.j2
 https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${NETWORK_ADDONS_VERSION}/network-addons-config.crd.yaml
 https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${NETWORK_ADDONS_VERSION}/namespace.yaml
 https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${NETWORK_ADDONS_VERSION}/operator.yaml
 https://github.com/MarSik/kubevirt-ssp-operator/releases/download/${SSP_VERSION}/kubevirt-ssp-operator-crd.yaml
-https://github.com/MarSik/kubevirt-ssp-operator/releases/download/${SSP_VERSION}/kubevirt-ssp-operator.yaml"
+https://github.com/MarSik/kubevirt-ssp-operator/releases/download/${SSP_VERSION}/kubevirt-ssp-operator.yaml
+https://github.com/kubevirt/vm-import-operator/releases/download/${VM_IMPORT_VERSION}/v2v.kubevirt.io_vmimportconfigs_crd.yaml
+https://github.com/kubevirt/vm-import-operator/releases/download/${VM_IMPORT_VERSION}/operator.yaml"
 
 OPERATOR_CRS="https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-cr.yaml
 https://github.com/kubevirt/containerized-data-importer/releases/download/${CDI_VERSION}/cdi-operator-cr.yaml
 https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${NETWORK_ADDONS_VERSION}/network-addons-config-example.cr.yaml
-https://github.com/MarSik/kubevirt-ssp-operator/releases/download/${SSP_VERSION}/kubevirt-ssp-operator-cr.yaml"
+https://github.com/MarSik/kubevirt-ssp-operator/releases/download/${SSP_VERSION}/kubevirt-ssp-operator-cr.yaml
+https://github.com/kubevirt/vm-import-operator/releases/download/${VM_IMPORT_VERSION}/v2v.kubevirt.io_vmimportconfigs_cr.yaml"
```

Comment:
the file was renamed to: vmimportconfig_cr.yaml

See
https://github.com/kubevirt/vm-import-operator/releases/download/v0.0.2/vmimportconfig_cr.yaml

Reply:
Done